qt/python/mantidqt/widgets/sliceviewer/peaksviewer/test/test_peaksviewer_presenter.py

```diff
@@ -67,13 +75,13 @@ def test_clear_removes_painted_peaks(self, mock_peaks_list_presenter):
         axes.get_xlim.return_value = (-1, 1)
         painter.axes = axes
         test_model.draw_peaks(slice_info, painter)
-        mock_view = MagicMock()
-        mock_view.painter = painter
-        presenter = PeaksViewerPresenter(test_model, mock_view)
+        self.mock_view = MagicMock()
```

Comment:
`self.mock_view` is created as an autospec mock object in `setUp`, so can remove subsequent lines of `self.mock_view = MagicMock()` that crop up in a few of the tests.